src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,41 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/cars/add-driver")
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/add-driver-to-car.jsp")
+                .forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        int carId = Integer.parseInt(req.getParameter("car_id"));
+        int driverId = Integer.parseInt(req.getParameter("driver_id"));
+        Car car = carService.get((long) carId);
+        Driver driver = driverService.get((long) driverId);
+        carService.addDriverToCar(driver, car);
+        req.getRequestDispatcher("/WEB-INF/views/add-driver-to-car.jsp")
+                .forward(req,resp);
```

Comment:
```suggestion
        resp.sendRedirect("/cars/add-driver")
```

Its better to use sendRedirect here